launch/nmea_driver_testing.launch

```diff
@@ -15,6 +15,8 @@
   <arg name="driver_rate"                 default="100" />
 
   <!-- Heading settings -->
+  <arg name="heading_as_imu"              default="False" />
```

Comment:
The name might indicate that is enables/disabled the IMU messages on all heading messages, so also the vendor specific ones, which isn't the case. Please either use a clearer name, indicating that this only applies to NMEA HDT messages or use it as switch for all heading IMU messages.

Reply:
Clarified by renaming the parameter to hdt_heading_as_imu.